compiler/modules/types/list.bal

```diff
@@ -257,6 +284,50 @@ function listInhabited(Context cx, SemType[] members, SemType rest, Conjunction?
     }
 }
 
+function shallowCopyListAtomicType(ListAtomicTypeMembers members) returns ListAtomicTypeMembers {
```

Comment:
Should have consistent naming pattern for functions (like this) operating on ListAtomicTypeMembers.

Reply:
Would `listMembersShallowCopy` work here? We already have `shallowCopyTypes` and `shallowCopyStrings`.